addon/components/pix-pagination.js

```diff
@@ -0,0 +1,146 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import { createIntl } from '@formatjs/intl';
+
+const DEFAULT_PAGE_OPTIONS = [
+  { label: '10', value: 10 },
+  { label: '20', value: 20 },
```

Comment:
J'ai l'impression que dans nos usages on devrait + avoir du 10-25-50-100 par défaut (et donc du custom côté page des tutos)
```suggestion
  { label: '25', value: 25 },
```